toolkit/tools/graphpkgfetcher/graphpkgfetcher.go

```diff
@@ -56,29 +56,65 @@ func main() {
 	kingpin.MustParse(app.Parse(os.Args[1:]))
 	logger.InitBestEffort(*logFile, *logLevel)
 
+	err := fetchPackages()
+	if err != nil {
+		logger.Log.Fatalf("Failed to fetch packages. Error: %s", err)
+	}
+}
+
+func fetchPackages() (err error) {
 	dependencyGraph, err := pkggraph.ReadDOTGraphFile(*inputGraph)
 	if err != nil {
-		logger.Log.Panicf("Failed to read graph to file. Error: %s", err)
+		err = fmt.Errorf("failed to read graph to file: %w", err)
```

Comment:
Related to our recent discussion, seemed to work fine when I tried out this approach:
```suggestion
		err = fmt.Errorf("failed to read graph to file:\n%w", err)
```